src/__tests__/get-local-project-data.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { getLocalProjectData } from '@/utils/get-local-project-data';
+
+const mockProjectJson = JSON.stringify([
+  {
+    title: 'Project 1',
+    description: 'Description 1',
+    links: [],
+    languages: ['JavaScript'],
+    status: 'In Progress',
+    location: 'City A',
+    contributors: ['User 1', 'User 2'],
+  },
+]);
+
+jest.mock('fs', () => ({
+  promises: {
+    readFile: jest.fn().mockResolvedValue(
+      JSON.stringify([
+        {
+          title: 'Project 1',
+          description: 'Description 1',
+          links: [],
+          languages: ['JavaScript'],
+          status: 'In Progress',
+          location: 'City A',
+          contributors: ['User 1', 'User 2'],
+        },
+      ])
```

Comment:
I cannot use mockProjectJson here because jest.mock() is automatically hoisted to the top of the file
More info: https://jestjs.io/docs/es6-class-mocks#calling-jestmock-with-the-module-factory-parameter